modules/oc-mirror-oci-format.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="oc-mirror-oci-format_{context}"]
+= Mirroring images in OCI format
+
+You can use the oc-mirror plug-in to mirror Operators in the Open Container Initiative (OCI) image format, instead of Docker v2 format. You can copy Operator images to a file-based catalog on disk in OCI format. Then you can then copy local OCI images to your target mirror registry.
+
+:FeatureName: Using the oc-mirror plug-in to mirror Operator images in OCI format
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have access to the internet to obtain the necessary container images.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `oc-mirror` CLI plug-in.
+
+.Procedure
+
+. Create the image set configuration file to retrieve the catalogs and images that you require.
++
+.Example image set configuration file for copying to disk
+[source,yaml]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v1alpha2
+mirror:
+ operators:
+ - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.11
+   packages:
+   - name: aws-load-balancer-operator
+----
++
+[NOTE]
+====
+When using the OCI feature, the `storageConfig` setting is ignored in favor of the location passed in to the `oc mirror` command.
+====
++
+// TODO: I removed "helm" from these examples, that okay?
+
+. Run the `oc mirror` command to mirror the images from the specified image set configuration to disk:
++
+[source,terminal]
+----
+$ oc mirror --config=./imageset-config.yaml \ <1>
+  --use-oci-feature \                         <2>
+  --oci-feature-action=copy \                 <3>
+  oci://my-oci-catalog                        <4>
+----
+<1> Pass in the image set configuration file. This procedure assumes that it is named `imageset-config.yaml`.
+<2> Use the `--use-oci-feature` flag to enable the OCI feature.
+<3> Set the `--oci-feature-action` flag to `copy` to copy the catalog to disk.
+<4> Specify a directory on disk where you want to output the catalog. This procedure assumes that it is named `my-oci-catalog`. The path must start with `oci://`. This directory is created in the current working directory where the `oc mirror` command is run.
++
+// TODO: Should this always be just a directory, or would a full path be okay?
++
+[NOTE]
+====
+You can optionally use the `--oci-registries-config` flag to specify the path to a TOML-formatted `registries.conf` file. You can use this to mirror from a different registry, such as a pre-production location for testing, without having to change the image set configuration file.
+
+.Example registries.conf file
+[source,toml]
+----
+[[registry]]
+ location = "production.registry.io:5000"
+ insecure = false
+ blocked = false
+ mirror-by-digest-only = true
+ prefix = ""
+ [[registry.mirror]]
+    location = "preprodlocation/test"
```

Comment:
After we know what we're putting for the production location, I'll want to update this to something similar, but just obvious that it's a pre-prod location

Reply:
@bergerhoffer - Yes  the "preprodlocation" just lets the admin know we are going to be using an alternative registry rather than the production registry , its obvious that it needs to be a viable registry URL

Reply:
you can keep preprodlocation, or if you prefer, use something like preprod-registry.mydomain.com

Reply:
Okay I updated the location to be `registry.redhat.io:5000` and the other one to be `preprod-registry.example.com`. Let me know if those as examples are okay, or if anything needs adjusted (like do we need a port on the pre-prod one?)